samples/Samples.HttpMessageHandler/Program.cs

```diff
@@ -105,6 +105,30 @@ private static async Task SendHttpClientRequestAsync(bool tracingDisabled)
                     }
                 }
             }
+
+#if NETCOREAPP
+            using (var client = new HttpClient(new SocketsHttpHandler()))
```

Comment:
Without the profiler changes, no span would be generated here because the only handler we even consider instrumenting is the `SocketsHttpHandler`, and before the changes we would not create a `Scope` for it